config/features.yml

```diff
@@ -565,22 +565,6 @@ features:
     actor_type: user
     description: After authentication redirects a user to the My VA page if they are on the homepage
     enable_in_development: true
-  rrd_mas_all_claims_tracking:
```

Comment:
The remaining `rrd_`-prefixed Flipper flags are also removed. Other than `rrd_call_vro_service`, none of these have been used by the code in months. See https://github.com/department-of-veterans-affairs/abd-vro/issues/1559 for the breakdown.